shared/FFTtrials.m

```diff
@@ -80,8 +80,8 @@
 
 if ~isempty(cfg.FFT.name)
     
-    cfg.LFP.name = cfg.FFT.name;
-    LFP = readLFP(cfg);
+%     cfg.LFP.name = cfg.FFT.name;
```

Comment:
Remove commented-out lines (but not in this case as previous point still needs to be resolved)